mozilla_bitbar_devicepool/configuration.py

```diff
@@ -70,8 +71,13 @@ def configure(bitbar_configpath, filespath=None, update_bitbar=False):
 
     logger.info('configure: starting configuration')
     start = time.time()
-    configure_device_groups(update_bitbar=update_bitbar)
-    configure_projects(update_bitbar=update_bitbar)
+    try:
+        configure_device_groups(update_bitbar=update_bitbar)
+        configure_projects(update_bitbar=update_bitbar)
+    except IOError as e:
+        logger.error("Test files seem to be missing! Please place and restart. Exiting...")
+        logger.error("%s: %s" % (e.__class__.__name__, e.message))
+        sys.exit(1)
```

Comment:
Let's also check that you are actually seeing the missing file exception. We shouldn't say we are missing the file if we aren't positive that is the problem.

Reply:
Via parsing the exception message or a custom exception lower in the stack?

Reply:
I'm working on raising and handling custom exceptions.